examples/window-app/common/src/WindowApp.cpp

```diff
@@ -430,7 +438,13 @@ void WindowApp::Cover::Init(chip::EndpointId endpoint)
     EndProductTypeSet(endpoint, EndProductType::kInteriorBlind);
 
     // Attribute: Id 24 Mode
-    Mode mode = { .motorDirReversed = 0, .calibrationMode = 1, .maintenanceMode = 1, .ledDisplay = 1 };
+    chip::BitFlags<Mode> mode;
+    mode.Clear(Mode::kMotorDirectionReversed);
+    mode.Clear(Mode::kMaintenanceMode);
+    mode.Clear(Mode::kCalibrationMode);
+    mode.Set(Mode::kLedFeedback);
```

Comment:
This doesn't match the old code, which set the maintenance/calibration bits.  Is this a purposeful change?

Reply:
Old code was wrong and should not set those at init.

Reply:
This really depend of the init state of a devices and it is manufacturer specific
For the vanilla demo everything should be ready to be used